src/styles/chat.css

```
@@ -0,0 +1,105 @@
+.chat_container {
+  display: grid;
+  grid-template-columns: 20% auto;
+  height: 100vh;
+  &__tape {
+    background: rgba(0, 123, 255, 0.05);
+    &__profile-link {
+      background: url('../static/img/profile_link_icon.png') right no-repeat;
+      display: flex;
+      justify-content: right;
+      border: none;
+      margin-left: auto;
+      margin-right: 30px;
+      font-family: 'Roboto';
+      font-style: normal;
+      font-weight: 400;
+      font-size: 18px;
+      line-height: 30px;
+      align-items: center;
+      color: #a5a0a0;
+      padding-right: 14px;
+    }
+  }
+  &__chats_list {
+  }
+  &__dialog {
+    padding: 5px 20px;
+    display: grid;
+    grid-template-rows: 5% auto 7%;
+    &__contact {
+      grid-row-start: 1;
+      grid-row-end: 2;
+      display: grid;
+      grid-template-columns: 7% 10% auto;
+      border-bottom: 1px solid #a5a0a0;
+      height: 60px;
+      &__avatar {
+        grid-column-start: 1;
+        grid-column-end: 2;
+        grid-row-start: 1;
+        grid-row-end: 2;
+        background: url('../static/img/Ellipse 1.png') no-repeat;
+        border-radius: 50%;
+        width: 50px;
+        height: 50px;
+        box-sizing: border-box;
+        display: block;
+        margin: auto;
+      }
+      &__name {
+        grid-column-start: 2;
+        grid-column-end: 3;
+        grid-row-start: 1;
+        grid-row-end: 2;
+        font-family: 'Roboto';
+        font-style: normal;
+        font-weight: 400;
+        font-size: 18px;
+        line-height: 30px;
+        color: #000000;
+        display: flex;
+        text-align: left;
+        align-items: center;
+      }
+      &__menu {
+        grid-column-start: 3;
+        grid-row-start: 1;
+        grid-row-end: 2;
+        background: url('../static/img/Group 3.png') no-repeat;
+        background-position: 0px 15px;
+        width: 30px;
+        height: 50px;
+        margin-left: auto;
+        margin-right: 0;
+      }
+    }
+    &__dialog {
+      grid-column-start: 1;
+      grid-column-end: 1;
+      grid-row-start: 2;
+      grid-row-end: 3;
+      padding: 20px 0px 5px 0px;
```

Сокращается
```suggestion
      padding: 20px 0px 5px;
```
https://developer.mozilla.org/en-US/docs/Web/CSS/Shorthand_properties#margin_and_padding_properties